src/loose/expression.js

```diff
@@ -514,8 +514,8 @@ lp.parseMethod = function(isGenerator, isAsync) {
     node.async = !!isAsync
   this.inAsync = node.async
   node.params = this.parseFunctionParams()
-  node.expression = this.options.ecmaVersion >= 6 && this.tok.type !== tt.braceL
```

Comment:
Was this supposed to be removed? I don't see where `.expression` is set now.

Reply:
Oh, I see, methods don't allow expressions anymore (they did in FF) so this code is obsolete indeed.